src/mixins/filing-template-mixin.ts

```diff
@@ -92,7 +92,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     switch (this.getEntityType) {
       case CorpTypeCd.COOP:
         filing.filing.incorporationApplication.cooperative = {
-          cooperativeAssociationType: this.getDefineCompanyStep.cooperativeType
+          cooperativeAssociationType: this.getDefineCompanyStep.cooperativeType,
+          rulesFileKey: this.getRules.docKey,
+          rulesFileName: this.getRules.rulesDoc ? this.getRules.rulesDoc.name : null
```

Comment:
Argus, you probably need some code under line 161 to restore the values when a draft is reloaded.

Reply:
This comment is handled by another comment :)